webserver/__init__.py

```diff
@@ -10,6 +10,10 @@ def create_postgres():
     from postgres_connection import init_postgres_connection
     return init_postgres_connection(current_app.config['SQLALCHEMY_DATABASE_URI'])
 
+def create_redis():
```

Comment:
> So you are doing that on every request to that endpoint? 
Yes.

> Why not do it once during app initialization process?
The same thing happens with the postgres connection as well. So to make things consistent i sticked with the same solution. Also what happens if we create a connection everytime?

> In fact, it's already being done: https://github.com/metabrainz/listenbrainz-server/blob/master/webserver/__init__.py#L32. Is there some kind of issue with that?
Nope. Actually we do have a persistent redis connnection as well with global name as `_redis`, its just the code is getting confusing. On some other branch i made those changes, so i did not want to create duplicates.